### PR TITLE
feat(FOROME-1403): dashboard switch button

### PR DESCRIPTION
**Pull request by Lapk1n**
[Task](https://quantori.atlassian.net/browse/FOROME-1403)
PR into separate [dashboard branch](https://github.com/ForomePlatform/Anfisa-React-Client/tree/feat/FOROME-1398)

### CODE REVIEW

File: src/components/units-list/units-list-controls/units-list-controls.tsx [hunk 1]
```diff
@@ -11,33 +12,62 @@ import { DecisionTreesResultsDataCy } from '@data-testid'
...
+    />
+
+    <button
+      className={styles.controls__button__listSize}
+      onClick={onToggleListSize}
+    >
+      <Icon name={isAllCollapsed ? 'Expand' : 'Collapse'} size={20} />
+    </button>
+
+    <button
```
Comment (AndreyUstyumenko) [inline src/components/units-list/units-list-controls/units-list-controls.tsx]: Create a component for this button and make a story for it.

**Review by AndreyUstyumenko — APPROVED** [review]

File: src/components/units-list/units-list-controls/units-list-controls.module.css [hunk 2]
```diff
@@ -7,12 +7,58 @@
     flex: 1 0 auto;
   }
 
-  &__button {
+  &__button__list-size {
```
Comment (EvgeniySobolev) [inline src/components/units-list/units-list-controls/units-list-controls.module.css]: If we use BEM notation it must look like `block__element_modifier`

File: src/components/units-list/units-list-controls/units-list-controls.module.css [hunk 3]
```diff
@@ -7,12 +7,58 @@
...
+
+    &__list {
+      position: relative;
+      right: -1px;
+      margin-left: 5px;
+      border-top-left-radius: 0.25rem;
+      border-bottom-left-radius: 0.25rem;
+    }
+
+    &__tile {
```
Comment (EvgeniySobolev) [inline src/components/units-list/units-list-controls/units-list-controls.module.css]: Are `tile` and `list` modifiers? Use single `_` pls

File: src/pages/filter/common/dashboard/components/dashboard-header.tsx [hunk 4]
```diff
@@ -8,41 +8,31 @@ import { t } from '@i18n'
...
   const [searchValue, setSearchValue] = useState('')
+  const { viewType, toggleViewType } = dashboardStore
 
   const onExpand = () => {
     //
   }
+
+  const handleSwitch = () => {
+    //
+  }
```
Comment (EvgeniySobolev) [inline src/pages/filter/common/dashboard/components/dashboard-header.tsx:24]: Remove?

**Review by EvgeniySobolev — APPROVED** [review]

**Pull request merged**